Containers/Home/Home.js

```diff
@@ -149,15 +149,15 @@ export default class Home extends Component {
                     })
                   }}
                 >
-                  <MapView.Callout tooltip={true} >
+                  <Callout tooltip={true} >
                     <View style={styles.landmarkInfo}>
                       <Text style={{ fontSize: 14, fontWeight: 'bold' }}>{location.name}</Text>
                       <Text style={{ fontSize: 14, fontStyle: 'italic' }}>{location.description}</Text>
                       <Text style={{ fontSize: 14 }}>Distance away: {this.state.calculatedDistance} mi</Text>
                       <Text style={{ fontSize: 14 }}>Visited: no</Text>
                       <Text style={{ fontSize: 14 }}>Link - takes to camera</Text>
                     </View>
-                  </MapView.Callout>
+                  </Callout>
                 </Marker>
               )
             })
```

Comment:
Are we going to still have Callouts now that we have the info displayed in the banner box above?

Reply:
Ya, I'm thinking we could use them as part of a tutorial when someone first opens the App. We could place a marker close to their location to have them walkthrough how the App works without having to go find the Markers far away.